lib/theme_check/checks/liquid_tag.rb

```diff
@@ -24,22 +29,46 @@ def on_tag(node)
     def on_string(node)
       # Only reset the counter on outputted strings, and ignore empty line-breaks
       if node.parent.block? && !node.value.strip.empty?
-        reset_consecutive_statements
+        reset_values
       end
     end
 
     def after_document(_node)
-      reset_consecutive_statements
+      reset_values
+      @first_nodes.each do |node|
+        add_offense("Use {% liquid ... %} to write multiple tags", node: node) do |corrector|
```

Comment:
Can we move the operations / expressions / statements in this block into methods to better describe what's going on?